pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -126,6 +126,24 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
   
+  <Target Name="PreserveSymbols"
+          AfterTargets="CreatePackage">
+    <PropertyGroup>
+      <_PackageSymbolsProjectDir>$(PackageSymbolsBinDir)$(MSBuildProjectName)/</_PackageSymbolsProjectDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <_SymbolsFiles Include="%(File.Identity)" Condition="'%(File.Extension)' == '.pdb' OR '%(File.Extension)' == '$(SymbolFileExtension)'">
+        <DestinationFolder>$(_PackageSymbolsProjectDir)%(File.TargetPath)/</DestinationFolder>
+      </_SymbolsFiles>
+      <_PackageSymbolsTargetFrameworkDir Include="@(_SymbolsFiles->'DestinationFolder')" />
+    </ItemGroup>
+    <MakeDir Directories="%(_SymbolsFiles.DestinationFolder)" />
+    <Message Importance="High" Text="Copy %(_SymbolsFiles.Identity) -> %(_SymbolsFiles.DestinationFolder)" />
```

Comment:
Is there a reason to do this instead of just using the output for the Copy task, which already writes a message?